### DISCUSSION TITLE
Please help: "Cuda out of memory"

### DISCUSSION THREAD

**Discussion by maiiabocharova** [facebookresearch/detectron2#3453] [Q&A]
Constantly getting "cuda out of memory" error.
```
cfg = get_cfg()
cfg.merge_from_file(model_zoo.get_config_file("COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml"))
cfg.DATASETS.TRAIN = ("experiment") 
cfg.DATASETS.TEST = ()
cfg.DATALOADER.NUM_WORKERS = 2
cfg.MODEL.WEIGHTS = model_zoo.get_checkpoint_url("COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml")
cfg.SOLVER.IMG_PER_BATCH = 1
cfg.SOLVER.BASE_LR = 0.00025
cfg.SOLVER.MAX_ITER = 1000
cfg.MODEL.ROI_HEADS.BATCH_SIZE_PER_IMAGE = 1
cfg.MODEL.ROI_HEADS.NUM_CLASSES = 10
```

I reduced IMG_PER_BATCH and BATCH_SIZE_PER_IMAGE to 1 and still I am getting this error.Please can you help me?

Upd: solved the issue by using updated code from tutorial

**Comment by ryanalexmartin** [comment]
Could you give a bit of information about your GPU, etc?

**Reply by maiiabocharova** [reply]
Hi, I am not facing this issue anymore: I used a current tutorial and it gave me no errors. Still don't know what was an issue with a bit older tutorial, but anyways
